features/step_definitions/direct_award_steps.rb

```diff
@@ -10,35 +15,23 @@
 
 When (/^I am ready to tell the outcome for the '(.*)' saved search$/) do |search_name|
   steps %{
-    And I have created and saved a search called '#{search_name}'
-    And I visit the /buyers page
-    Then I click the 'View your saved searches' link
-    Then I click the '#{search_name}' link
-    Then I am on the '#{search_name}' page
-    And I click the 'Export your results' link
-    Then I am on the 'Before you export your results' page
-    And I check 'I understand that I cannot edit my search again after I export my results' checkbox
-    And I click the 'Export results and continue' button
-    Then I am on the 'my cloud project' page
-    When I have downloaded the search results as a file of type 'ods'
+    Given I have created and saved a search called '#{search_name}'
+    And I am on the '#{search_name}' page
+    And I have exported my results for the '#{search_name}' saved search
     And I click the 'Return to your task list' link
```

Comment:
This line seems to be causing a few failures - clicking the export link goes straight back to the task list (is it meant to do this?).